package.json

```diff
@@ -8,6 +8,7 @@
     "files": [
       "test/**/*.test.js"
     ],
+    "failFast": true,
```

Comment:
i think this will help our situation where we have tests failing/timing out but not exiting in our pipeline. it does make the issue of not cleaning up after the integration test a bit worse, but that appears to be an issue with [how `test.after.always` works in ava](https://github.com/avajs/ava/issues/928) and already existed in some other situations. that should only be a problem with local test executions.